### PR TITLE
Really fix missing JSON data.

### PR DESCRIPTION
**Pull request by mjfrey**
Don't get fancy and check if JSON attributes exist before trying to get them.

### CODE REVIEW

**Comment by mjfrey** [comment]
@douglashall

File: ecommerce/extensions/basket/views.py [hunk 1]
```diff
@@ -107,7 +107,12 @@ def get_context_data(self, **kwargs):
             short_description = None
             try:
                 course = get_course_info_from_lms(course_key)
-                image_url = course.get('media', '{}').get('image', '{}').get('raw', '')
+                image = course.get('media')
+                if image:
+                    image = image.get('image')
+                    if image:
+                        image = image.get('raw', '')
+                image_url = image
```
Comment (douglashall) [inline ecommerce/extensions/basket/views.py]: Should `image_url` get set even if you don't fall all the way through the if hierarchy above?

For instance, do you ever want `image_url` to get set to the result of `course.get('media')`?

Comment (douglashall) [inline ecommerce/extensions/basket/views.py]: I would favor something like:

```
try:
    image_url = course['media']['image']['raw']
except (KeyError, TypeError):
    image_url = ''
```

**Comment by douglashall** [comment]
👍 Code looks good. You could probably use some tests to verify the exceptions catch on image lookup if your timetable permits.

**Pull request merged**